### PR TITLE
Blog styles

### LINKED ISSUE #195: blog content styles not working.
- some styles for blog content are not being applied correctly eg Headings h1, h2, etc. not displaying in the correct text size or weight (boldness).

syt

### PR DESCRIPTION
**Pull request by bugswritter**
[Have you read the contributing guidelines ?](https://github.com/SpaceyaTech/SYT-Web-Redesign/blob/Dev/docs/CONTRIBUTING.md)

# What is the purpose of your *pull request*?
 - [x] Bug fix
 - [ ] New feature

- Fixes #195
# Proposed changes
- Replaced predefined CSS classes for blog content styling with Tailwind CSS Typography classes.
- This change ensures consistent and responsive typography styling across the blog content.
- I also updated the useEffect hook in the BlogBody component to set the innerHTML of the blog content container directly instead of appending to it, which prevents duplicate content from being rendered when the blogBody prop changes.
- Styling the First Letter: The first letter of each blog post is now styled to be bold and capitalized 
- Changing Text Color to Darker Shade for better readability and consistency across all blog posts.

# Issues I Encountered.

- HTML Structure Variations from Content Editors: Some blog content uses incorrect HTML tags for formatting. For example: `<p> ` tags are being used for headings instead of semantic tags like `<h2> <h3>` etc. [blog](https://www.spaceyatech.com/blogs/why-ux-research-is-important-and-how-to-do-it-the-right-way-in-2024). Which leads to Tailwind’s typography classes (prose) not being fully effective, as they rely on proper HTML semantics.
- Global Font Family Override:  ` * {
  font-family: "Poppins", sans-serif;
}` applies the "Poppins" font family to all elements, including blog content. Even when the prose classes are applied, making the blog body look more bold.

# Warning

# Please read these points carefully and answer honestly with an `X`
into all the boxes. Example : [X] 

Before submitting a *pull request* make sure you have:

 - [x] Read the guidelines for contributing.
 - [ ] Wrote some tests.
 - [x] Respected the linting guidelines.

### CODE REVIEW

File: src/pages/blog2/sections/BlogBody.jsx [hunk 1]
```diff
@@ -7,23 +7,23 @@ import { glovo } from "../../../assets/images/blogs-page";
...
   return (
     <div className="flex flex-col md:flex-row gap-5 w-full pb-8 px-3">
       <div className="w-full md:w-[70%] flex flex-col gap-5">
         <div className="container text-[13px] md:text-base font-normal text-[#323433]">
-          <div ref={BodyRef} className="blog-content" />
-
+          <div
+            ref={BodyRef}
+            className="prose-lg prose prose-a:no-underline  prose-li:marker:text-[#333] prose-li:marker:text-sm prose-a:text-[#009975]"
+          />
```
Comment (sonylomo) [inline src/pages/blog2/sections/BlogBody.jsx]: The text styles are inconsistent with the [original Figma Designs](https://www.figma.com/design/QZwtseXogSFjLNz64uLwjV/SYT-Website-Redesign-Complete?node-id=6943-12373&m=dev) i.e it's too small for a blog post and the color is too light.

Comment (sonylomo) [reply src/pages/blog2/sections/BlogBody.jsx]: Thanks for adding the typography, that was a good call.

**Review by sonylomo — CHANGES_REQUESTED** [review]

**Comment by JimmyTron** [comment]
there's a merge conflict in the offing here.

**Comment by sonylomo** [comment]
Hey @Joe-Mwangi 👋🏾 

Are you still working on this?

**Comment by bugswritter** [comment]
> Hey @Joe-Mwangi 👋🏾
> 
> Are you still working on this?

Yes am working on it. Thanks for the reminder

**Review by JimmyTron — APPROVED** [review]
LGTM 👌

**Review by Ase020 — APPROVED** [review]

**Review by sonylomo — APPROVED** [review]

**Pull request merged**